code/modules/keybindings/bindings_client.dm

```diff
@@ -47,24 +47,17 @@
 	// Client-level keybindings are ones anyone should be able to do at any time
 	// Things like taking screenshots, hitting tab, and adminhelps.
 
-	switch(_key)
-		if("F1")
-			if(keys_held["Ctrl"] && keys_held["Shift"]) // Is this command ever used?
-				winset(src, null, "command=.options")
-			else
-				get_adminhelp()
-			return
-		if("F2") // Screenshot. Hold shift to choose a name and location to save in
-			winset(src, null, "command=.screenshot [!keys_held["shift"] ? "auto" : ""]")
-			return
-		if("F12") // Toggles minimal HUD
-			mob.button_pressed_F12()
-			return
+	var/AltMod = keys_held["Alt"] ? "Alt-" : ""
+	var/CtrlMod = keys_held["Ctrl"] ? "Ctrl-" : ""
+	var/ShiftMod = keys_held["Shift"] ? "Shift-" : ""
+	var/full_key = "[AltMod][CtrlMod][ShiftMod][_key]"
+	for (var/kb_name in prefs.key_bindings[full_key])
+		var/datum/keybinding/kb = GLOB.keybindings_by_name[kb_name]
+		if(kb.down(src))
```

Comment:
I feel like we should be allowing multiple things to go off one key if that's how they configured it, just put some sane maximum at like 5 or something.

Reply:
don't think the system supports multi-key bindings if that's what you mean

Reply:
added a cap of 5 commands on one key